.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    parserOptions: {
+      ecmaVersion: 2019,
+      sourceType: "module",
+    },
+    env: {
+      browser: true,
+      es6: true,
+    },
+    extends: ["airbnb-base"],
+    plugins: ["jest", "svelte3"],
+    rules: {
+      "import/prefer-default-export": "off",
+      "import/no-extraneous-dependencies": ["error", { devDependencies: true }],
+      "no-param-reassign": [
+        "error",
+        { props: true, ignorePropertyModificationsFor: ["draft"] },
+      ],
+    },
```

Comment:
Do these rules need to be modified? I like them, but it may be least controversial to just stick to airbnb rules. Otherwise, we're inventing a new ruleset: airbnb + graph-paper-org defaults